games-puzzle/blockout2/blockout2-2.5.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="3D falling blocks game"
+DESCRIPTION="BlockOut II is an OpenbGL adaptation of the original BlockOut DOS \
+game edited by California Dreams in 1989. BlockOut II has the same features \
+than the original game with few graphic improvements. Score calculation is also \
+nearly similar to the original Game. BlockOut II has been designed by an \
+addicted player for addicted players."
+HOMEPAGE="http://www.blockout.net/blockout2/"
+COPYRIGHT="2007-2014 Jean-Luc Pons"
+LICENSE="GNU GPL v2"
+REVISION="1"
+
+SOURCE_URI="https://salsa.debian.org/games-team/blockout2/-/archive/master/blockout2-master.tar.bz2"
+CHECKSUM_SHA256="08b677be01c02e6695559ccc65df1cafcac0793b7864ba7e2285059babf57349"
+SOURCE_DIR="blockout2-master/"
+
+PATCHES="
+	blockout2-2.5.patchset
+	"
```

Comment:
Some small changes, no new empty lines, trailing slash in SOURCE_DIR shouldn't be needed, keep PATCHES on one line:
```diff
-REVISION="1"
-
-SOURCE_URI="https://salsa.debian.org/games-team/blockout2/-/archive/master/blockout2-master.tar.bz2"
-CHECKSUM_SHA256="08b677be01c02e6695559ccc65df1cafcac0793b7864ba7e2285059babf57349"
-SOURCE_DIR="blockout2-master/"
-
-PATCHES="
-	blockout2-2.5.patchset
-	"
+REVISION="1"
+SOURCE_URI="https://salsa.debian.org/games-team/blockout2/-/archive/master/blockout2-master.tar.bz2"
+CHECKSUM_SHA256="08b677be01c02e6695559ccc65df1cafcac0793b7864ba7e2285059babf57349"
+SOURCE_DIR="blockout2-master"
+PATCHES="blockout2-2.5.patchset"
```